doc/src/guides/solving.md

```diff
@@ -0,0 +1,30 @@
+# Solve Using the SymPy Python Library
+
+The Python package SymPy can symbolically solve equations, differential equations, 
+linear equations, nonlinear equations, matrix problems, inequalities, 
+diophantine equations, and evaluate integrals. SymPy can also solve numerically.
+
+Learn how to use SymPy computer algebra system to:
+
+| Description                                                  | Example                                                                                                                     |
+|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| [ Solve an equation algebraically ](http://docs.sympy.org)                        | [  $x^2 - 1 = 0$  ](http://docs.sympy.org)                                                                                                       |
+| [ Solve a system of equations algebraically ](http://docs.sympy.org)              | [  $x^2 + y = 1, x - y = 0$  ](http://docs.sympy.org)                                                                                              |
+| [ Solve an equation numerically ](http://docs.sympy.org)                          | [ $x^2 + x = 0.274$ ](http://docs.sympy.org)                                                                                                     |
```

Comment:
I would suggest using an equation that doesn't have a closed-form solution (the example I like to use is cos(x) = x)

Reply:
Great, I just used that example in my latest commit. (Are we sure HackerRank hasn't copyrighted that example? ;-)